specificatie/openapi.yaml

```diff
@@ -1138,8 +1153,13 @@ components:
     VerblijfplaatsBeperkt:
       type: object
       description: |
-                      Gegevens over het verblijf of de woonlocatie van een persoon.
+                    Gegevens over het verblijf of de woonlocatie van een persoon.
+                    * **functieAdres: wordt gevuld op basis van de waarden die voorkomen in de tabel 'soortadres' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
ik zou dat "Soort_Adres" noemen, in lijn met hoe de andere tabellen genoemd zijn

Reply:
Aangepast